src/sphinx_book_theme/assets/styles/_footer.scss

```diff
@@ -2,9 +2,9 @@
 * Footer *
 *********************************************/
 footer {
-    padding-left: 0;
```

Comment:
Just to confirm - have you changed anything in the CSS files other than either moving them or updating their formatting in a way that doesn't change end behavior? If you *have* changed something, can you briefly describe what you've changed?

Reply:
I have added a few styles, for the margin directive in this commit https://github.com/executablebooks/sphinx-book-theme/pull/469/commits/4fb51549f59c6075ba5e1eb185ded991a244031e , which I pointed out in this comment https://github.com/executablebooks/sphinx-book-theme/pull/469#issuecomment-1019616614. Still need to find the issue which was created for this. I will update that issue once I find it.